src/main/java/core/screens/HUD/HudManager.java

```diff
@@ -0,0 +1,108 @@
+package core.screens.HUD;
+
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.graphics.Color;
+import com.badlogic.gdx.graphics.OrthographicCamera;
+import com.badlogic.gdx.graphics.g2d.BitmapFont;
+import com.badlogic.gdx.graphics.g2d.SpriteBatch;
+import com.badlogic.gdx.scenes.scene2d.Stage;
+import com.badlogic.gdx.scenes.scene2d.ui.Label;
+import com.badlogic.gdx.scenes.scene2d.ui.Table;
+import com.badlogic.gdx.utils.Disposable;
+import com.badlogic.gdx.utils.viewport.FitViewport;
+import com.badlogic.gdx.utils.viewport.Viewport;
+
+public class HudManager implements Disposable {
```

Comment:
possibly rename to GameOverlayManager or similar, if we're gonna handle more things than just HUD which need to be static on the screen

for example lil popups, point/cross where ur aiming, inventory & other windows, etc

Reply:
Overall, I think it might make more sense to either just have an area in GameplayController(GameScreen) for drawing things in the camera's frame of reference, or having a specific GameOverlayController for that

then just have separate classes for HUD, InventoryWindow, Etc